src/mocks/data/journalResp.json

```diff
@@ -1,180 +1 @@
5:00+00:00\",\"duration\":57,\"vehicleSlotType\":\"B57mins\",\"_testCentreId\":54321,\"testCentreName\":\"Example Test Centre\"},\"booking\":{\"candidate\":{\"_candidateId\":100,\"candidateName\":{\"title\":\"Miss\",\"firstName\":\"Ammar\",\"secondName\":\"Bbb\",\"thirdName\":\"Ccc\",\"lastName\":\"Haider\"},\"driverNumber\":\"AAAAA123456A78AA\",\"gender\":\"Female\",\"candidateAddress\":{\"line1\":\"1 Some Street\",\"line2\":\"Some place\",\"line3\":\"Some area\",\"line4\":\"Some town\",\"line5\":\"Some county\",\"postcode\":\"AA1 2BB\"},\"primaryTelephone\":\"01234 567890\",\"secondaryTelephone\":\"04321 098765\",\"mobileTelephone\":\"07654 123456\",\"email\":\"aaaa@bbbb.com\",\"prn\":11223344,\"previousADITests\":2},\"application\":{\"_applicationId\":\"384 392 293 22\",\"_bookingSequence\":\"03\",\"_checkDigits\":\"01\",\"welshTest\":false,\"extendedTest\":true,\"meetingPlace\":\"Some meeting place\",\"progressiveAccess\":false,\"specialNeeds\":\"Heavily pregnant\",\"checkMarker\":false,\"vehicleSeats\":5,\"vehicleHeight\":1.75,\"vehicleWidth\":2.5,\"vehicleLength\":5.5,\"testCategory\":\"B+E\"},\"previousCancellations\":[\"Act of nature\",\"DVSA\",\"DSA\"],\"business\":{\"_id\":1000,\"name\":\"Dave's Dodgy Driving Instructors\",\"businessAddress\":{\"line1\":\"Flat 1A\",\"line2\":\"1 Example Street\",\"line3\":\"Example Area\",\"line4\":\"Example Town\",\"line5\":\"Example County\",\"postcode\":\"AB12 3CD\"},\"telephone\":\"07890 654321\"}}},{\"slot\":{\"_slotId\":1001,\"start\":\"2018-03-05T14:32:00+00:00\",\"duration\":57,\"vehicleSlotType\":\"B57mins\",\"_testCentreId\":54321,\"testCentreName\":\"Example Test Centre\"},\"booking\":{\"candidate\":{\"_candidateId\":100,\"candidateName\":{\"title\":\"Miss\",\"firstName\":\"Leigh\",\"secondName\":\"Bbb\",\"thirdName\":\"Ccc\",\"lastName\":\"White\"},\"driverNumber\":\"AAAAA123456A78AA\",\"gender\":\"Female\",\"candidateAddress\":{\"line1\":\"1 Some Street\",\"line2\":\"Some place\",\"line3\":\"Some area\",\"line4\":\"Some town\",\"line5\":\"Some county\",\"postcode\":\"AA1 2BB\"},\"primaryTelephone\":\"01234 567890\",\"secondaryTelephone\":\"04321 098765\",\"mobileTelephone\":\"07654 123456\",\"email\":\"aaaa@bbbb.com\",\"prn\":11223344,\"previousADITests\":2},\"application\":{\"_applicationId\":\"384 392 293 22\",\"_bookingSequence\":\"03\",\"_checkDigits\":\"01\",\"welshTest\":false,\"extendedTest\":true,\"meetingPlace\":\"Some meeting place\",\"progressiveAccess\":false,\"specialNeeds\":\"Heavily pregnant\",\"checkMarker\":false,\"vehicleSeats\":5,\"vehicleHeight\":1.75,\"vehicleWidth\":2.5,\"vehicleLength\":5.5,\"testCategory\":\"B+E\"},\"previousCancellations\":[\"Act of nature\",\"DVSA\",\"DSA\"],\"business\":{\"_id\":1000,\"name\":\"Dave's Dodgy Driving Instructors\",\"businessAddress\":{\"line1\":\"Flat 1A\",\"line2\":\"1 Example Street\",\"line3\":\"Example Area\",\"line4\":\"Example Town\",\"line5\":\"Example County\",\"postcode\":\"AB12 3CD\"},\"telephone\":\"07890 654321\"}}}]}}"}
```

Comment:
Do we want to do this? Looks hard to edit now!

Reply:
No we don't, but this is the exact format of the response you get from the microservice. Now that the app is integrated it needs to follow the same format, I'll create a task to clean this up in both repo's at a later time..